releasenotes/notes/add-pvc-as-tag-3ec2aa03565a3a4d.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add persistent volume claim as tag (`persistentvolumeclaim:<pvc_name>`) to StatefulSets pods.
```

Comment:
it's RST format you want double back-quote there http://docutils.sourceforge.net/docs/user/rst/quickstart.html